util/hookable/hookable.cpp

```diff
@@ -342,6 +404,115 @@ J9HookRegister(struct J9HookInterface **hookInterface, uintptr_t taggedEventNum,
 					(emptyRecord->next->agentID >= agentID)))
 		) {
 			emptyRecord->function = function;
+			emptyRecord->callsite = NULL;
```

Comment:
Right  - so why not have J9HookRegister forward to J9HookRegisterWithCallSite?

```
J9HookRegister(...) {
   return J9HookRegisterWithCallSite(......, NULL);
}
```
So you only have 1 implementation?